lib/modules/minifyCss.es6

```diff
@@ -28,9 +28,13 @@ export default function minifyCss(tree, options, cssnanoOptions) {
 function processStyleNode(styleNode, cssnanoOptions) {
     let css = extractCssFromStyleNode(styleNode);
 
-    const strippedCss = stripCdata(css);
-    const isCdataWrapped = css !== strippedCss;
-    css = strippedCss;
+    // Improve performance by avoiding calling stripCdata again and again
+    let isCdataWrapped = false;
+    if (css.includes('CDATA')) {
+        const strippedCss = stripCdata(css);
+        isCdataWrapped = css !== strippedCss;
+        css = strippedCss;
+    }
```

Comment:
By testing, if `css` includes `CDATA`, most of unnecessary `strippedCss()` can be avoided.